### PR TITLE
[feat] User Auth: Created a simple login page that supports sign up and sign in.

### PR DESCRIPTION
**Pull request by rachaelch3n**
### Description
Create a very simple login page that supports sign up and sign in.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ccatherinetan

### CODE REVIEW

File: app/(auth)/login/page.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign up: ${error}`);
+    }
+
+    return data;
+  };
+
+  const signInWithEmail = async () => {
```
Comment (ccatherinetan) [inline app/(auth)/login/page.tsx]: let's rename this to `handleSignInWithEmail` for consistency 
Can we also reroute to the home page? (`/`) to confirm that the user has logged in for testing purposes? You can route to another page like [this](https://nextjs.org/docs/app/api-reference/functions/use-router)

```
import { useRouter } from 'next/navigation';
...
// in your component 
const { push } = useRouter();

// in your function 
push("/");
```

File: app/(auth)/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      <input
+        type="password"
+        name="password"
+        onChange={e => setPassword(e.target.value)}
+        value={password}
+      />
+      <button type="button" onClick={handleSignUp}>
+        Sign up
+      </button>
+      <button type="button" onClick={signInWithEmail}>
```
Comment (kevin3656) [inline app/(auth)/login/page.tsx]: I think this looks good, maybe could put some identifier for where the email goes and where the password goes for signup/login

Comment (ccatherinetan) [reply app/(auth)/login/page.tsx]: good point! maybe consider creating a placeholder for each of the text inputs 
e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder
`<input placeholder="Email" />` and `<input placeholder="Password" />`

**Review by ccatherinetan — COMMENTED** [review]
looking good! even tho we don't have the final designs, it would be nice to have some way to confirm that sign up / sign in has worked. 
1. for now, maybe u can route the user to home page after clicking either button
2. maybe add a placeholder to both inputs to distinguish them for now. 
3. you can consider creating some general styles (e.g. centering each input/button etc) to practice Styled Components!

**Comment by SashankBalusu** [comment]
Apart from the previous comments it looks good and tests great for me!

**Review by ccatherinetan — APPROVED** [review]
looks great! thanks rachael! let's merge it in today

**Pull request merged**